client/app/bundles/shared/components/Switch.scss

```diff
@@ -0,0 +1,64 @@
+@import "_legacy.scss";
+@import "_base.scss";
+
+.switch {
+  position: relative;
+  display: inline-block;
+  width: 119px;
+  height: 49px;
+}
+
+.switch input {
+  position: absolute;
+  top: -99999px;
+  left: -99999px;
+}
+
+.slider {
+  position: absolute;
+  font-size: $small-font;
+  font-family: $font-family;
+  text-align: center;
+  vertical-align: middle;
+
+  line-height: 40px;
+  color: $mulberry-wood;
+  letter-spacing: 9.44%;
+
+  cursor: pointer;
+  top: 0; left: 0; right: 0; bottom: 0;
+  background-color: $mulberry-wood;
+  border: 0px solid rgba(255, 255, 255, 0.7);
+  border-radius: 5px;
+  -webkit-transition: .2s;
+  transition: .2s;
+}
+
+input:checked + .slider {
+  background-color: #BACF88;
+}
+
+input:focus + .slider {
+  box-shadow: 0 0 1px #BACF88;
+}
+
+.slider:after {
+  position: absolute;
+  content: "OFF";
```

Comment:
Let's pass in an i18n key here

Reply:
How do I pass an i18n key here? I'm unable to understand. I tried using attribute.